src/commands/dev.rs

```diff
@@ -125,7 +125,9 @@ impl Dev {
 
         for path in paths_to_watch {
             let path = PathBuf::from(path);
-            watcher.watch(path.as_path(), notify::RecursiveMode::Recursive)?;
+            if path.exists() {
+                watcher.watch(path.as_path(), notify::RecursiveMode::Recursive)?;
+            }
```

Comment:
do we want to give a message if it doesn't that helps explain what might be happening?

Reply:
not sure how helpful that would be, imagine you have a minimal oranda setup (no config file, no book, etc), and you immediately get like 3 warning log messages. since the user doesn't add these paths i think it's fine to silently ignore

Reply:
ah, so i was not fully understanding what was happening here. now that i do - i can make my comment more specific, which is... do we warn/error if there are no paths being watched? (is it possible that no path can be watched?)

Reply:
yes, if the user doesn't even have a readme file, although i think oranda will error anyways in that case